library-best-practices/JLBP-14.md

```diff
@@ -0,0 +1,11 @@
+[JLBP-14] Use explicit versions, not version ranges
+---------------------------------------------------
+
+- Neither Maven nor Gradle have have a mechanism to locally lock versions like
+  package managers in other languages. If the pom.xml of your library specifies
+  a version range instead of an explicit version, it can pull in snapshot
```

Comment:
I'm not sure it will get a snapshot version in this case.

Reply:
Ok I wasn't sure myself - I think this statement was written by someone else and rode along to my PR. I'll just remove the reference to snapshots.